# Constants
int8 STANDARD = 1
int8 EMERGENT = 4
# Fields
int8 odlc_type